From: Jeremy Whiting <jpwhiting@kde.org>
Date: Thu, 27 Aug 2015 02:11:03 +0000
Subject: Allow selecting both files and folders in kompare's inital dialog.
X-Git-Url: http://quickgit.kde.org/?p=kompare.git&a=commitdiff&h=5ddaf63d1a7ee24b3fe8a4d102b192d40c4c7fc9
---
Allow selecting both files and folders in kompare's inital dialog.

BUG:351782
---


--- a/main.cpp
+++ b/main.cpp
@@ -228,8 +228,8 @@
 
 			dialog.setGroup( "Recent Compare Files" );
 
-			dialog.setFirstURLRequesterMode( KFile::File|KFile::Directory|KFile::ExistingOnly );
-			dialog.setSecondURLRequesterMode( KFile::File|KFile::Directory|KFile::ExistingOnly );
+			dialog.setFirstURLRequesterMode( KFile::File|KFile::ExistingOnly );
+			dialog.setSecondURLRequesterMode( KFile::File|KFile::ExistingOnly );
 
 			if( dialog.exec() == QDialog::Accepted )
 			{

